frontend/pages/hosts/details/cards/Software/SoftwareDetailsModal/SoftwareDetailsModal.tsx

```diff
@@ -88,69 +91,96 @@ const SoftwareDetailsInfo = ({
 };
 
 interface ISoftwareDetailsModalProps {
+  hostDisplayName: string;
   software: IHostSoftware;
   onExit: () => void;
 }
 
-const SoftwareDetailsModal = ({
+const SoftwareDetailsContent = ({
   software,
-  onExit,
-}: ISoftwareDetailsModalProps) => {
-  const installUuid = software.last_install?.install_uuid || "";
-
-  const renderSoftwareDetails = () => {
-    const { installed_versions } = software;
-
-    // special case when we dont have installed versions. We can only show the
-    // software type atm.
-    if (!installed_versions || installed_versions.length === 0) {
-      return (
-        <div className={`${baseClass}__software-details`}>
-          <DataSet
-            title="Type"
-            value={formatSoftwareType({ source: software.source })}
-          />
-        </div>
-      );
-    }
+}: Pick<ISoftwareDetailsModalProps, "software">) => {
```

Comment:
is this to intentionally couple this with `ISoftwareDetailsModalProps`? If not what about just `IHostSoftware`?

Reply:
Yes, that was the intention